Automattic/wp-calypso
# Plans: Add feature descriptions

Fixes #6568.

## Testing Instructions

1. Run Calypso with `ENABLE_FEATURES=manage/plan-features make run`;
2. Go to `/plans`;
3. See the popup feature descriptions;
4. Read the copy and make sure it matches @apeatling's one in #6568 and/or is typo-free.

## Screenshots

![selection_034](https://cloud.githubusercontent.com/assets/4988512/16630123/45ee332c-43b9-11e6-928b-95a198187d9a.png)

## Questions/notes for @apeatling 

We don't have "Basic Design Customization", "Remove WordPress.com Branding" features yet, therefore, I'm not adding the description. Is that okay?

I have not changed the text for "$200 Advertising Voucher" because currently it includes a nice divider. Should I add the new copy and update the divider?

![selection_035](https://cloud.githubusercontent.com/assets/4988512/16632814/498278b0-43c6-11e6-8c2f-9ee14e36a71e.png)

```js

getDescriptionWithWordAdsCredit: () => i18n.translate( '$100 Google AdWords credit after spending the first $25. Offer valid in US and Canada. {{hr/}}Business also includes $100 of advertising from WordAds on WordPress.com.', {
	components: {
		hr: <hr className="plans-compare__info-hr"/>
	}
} )
```

@gwwar 



Test live: https://calypso.live/?branch=add/plan-features-descriptions
